automation/patch_igz/patch_remote.py

```diff
@@ -274,6 +281,121 @@ def _wait_deployment_ready(self):
             live=True,
         )
 
+    def _reset_mlrun_db(self):
+        logger.info("Scaling down mlrun-api-chief")
+        self._exec_remote(
+            [
+                "kubectl",
+                "-n",
+                "default-tenant",
+                "scale",
+                "deploy",
+                "mlrun-api-chief",
+                "--replicas=0",
+            ],
+        )
+        logger.info("Scaling down mlrun-api-worker")
+        self._exec_remote(
+            [
+                "kubectl",
+                "-n",
+                "default-tenant",
+                "scale",
+                "deploy",
+                "mlrun-api-worker",
+                "--replicas=0",
+            ],
+        )
+
+        logger.info("Waiting for mlrun-api-chief to go down")
+        self._exec_remote(
+            [
+                "kubectl",
+                "-n",
+                "default-tenant",
+                "wait",
+                "pods",
+                "-l",
+                "app.kubernetes.io/sub-component=chief",
+                "--for=delete",
+                "--timeout=60s",
+            ],
+            live=True,
+        )
+
+        logger.info("Waiting for mlrun-api-worker to go down")
+        self._exec_remote(
+            [
+                "kubectl",
+                "-n",
+                "default-tenant",
+                "wait",
+                "pods",
+                "-l",
+                "app.kubernetes.io/sub-component=worker",
+                "--for=delete",
+                "--timeout=60s",
+            ],
+            live=True,
+        )
+
+        mlrun_db_pod = self._get_db_pod()
+        if mlrun_db_pod is None:
+            raise RuntimeError("Unable to find DB pod")
+
+        logger.info("reset DB")
+        self._exec_remote(
+            [
+                "kubectl",
+                "-n",
+                "default-tenant",
+                "exec",
+                "-it",
+                mlrun_db_pod,
+                "--",
+                "mysql",
+                "-u",
+                self._config["DB_USER"],
+                "-S",
+                "/var/run/mysqld/mysql.sock",
+                "-e",
+                "'DROP DATABASE mlrun; CREATE DATABASE mlrun'",
+            ],
+            live=True,
+        )
+
+        logger.info("Scaling up mlrun-api-chief")
+        self._exec_remote(
+            [
+                "kubectl",
+                "-n",
+                "default-tenant",
+                "scale",
+                "deploy",
+                "mlrun-api-chief",
+                "--replicas=1",
+            ],
+        )
+        logger.info("Scaling up mlrun-api-worker")
+        self._exec_remote(
+            [
+                "kubectl",
+                "-n",
+                "default-tenant",
+                "scale",
+                "deploy",
+                "mlrun-api-worker",
+                "--replicas=1",
```

Comment:
What if there were more than 1 workers? What if there were no workers (only chief is deployed)? 
Should check how many replicas were before, and return to that situation. Alternatively - let the user enter this as a config to the script (could also be used as a different command just to scale-up/down).